File: gui_view/src/main/scala/com/goexp/galgame/gui/view/game/listview/simplelist/small/CellController.java
package com.goexp.galgame.gui.view.game.listview.simplelist.small;

import com.goexp.galgame.gui.model.Game;
import com.goexp.galgame.gui.view.DefaultController;
import javafx.fxml.FXML;

public class CellController extends DefaultController {

    @FXML
    private HeaderController headerController;


    protected void initialize() {


        //        imageImg.setOnMouseEntered(event -> {
        //            rightPanel.setVisible(true);
        //        });
        //
        //        imageImg.setOnMouseExited(event -> {
        //            rightPanel.setVisible(false);
        //        });
    }


    public void load(Game game) {

        headerController.load(game);
    }

}

File: gui_view/src/main/scala/com/goexp/galgame/gui/view/game/listview/simplelist/small/CellController.scala
package com.goexp.galgame.gui.view.game.listview.simplelist.small

import com.goexp.galgame.gui.model.Game
import com.goexp.galgame.gui.view.DefaultController
import javafx.fxml.FXML

class CellController extends DefaultController {
  @FXML private var headerController: HeaderController = _

  override protected def initialize() = {
  }

  def load(game: Game) = headerController.load(game)
}